module/Photo/src/Photo/Model/VirtualAlbum.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+namespace Photo\Model;
+
+/**
+ * VirtualAlbum.
+ * Album that will never be stored in the database as such.
+ */
+class VirtualAlbum extends Album
+{
+    /**
+     * Album ID.
+     *
+     */
+    protected $id;
+    
+    /**
+     * First date of photos in album
+     */
+    protected $startDateTime = null;
```

Comment:
@jszanto Would it be better to remove these variables? They can be inherited from the Album class, but there they have ORM annotations and here the explicitly do not.

Reply:
The ORM notations won’t be inherited, so it would be better to remove them.

Reply:
I have removed it now.